examples/elmoControl/include/CanOpenErrors.h

```diff
@@ -0,0 +1,88 @@
+#ifndef CAN_ELMO_ERROR_H
+#define CAN_ELMO_ERROR_H
+
+#include <cstdint>
+#include <cstdio>
+
+namespace can
+{
+    namespace emergency
```

Comment:
too many namespaces and not meaningful. BTW the first namespace shall be kickcat